src/content/protocols/aave.md

```diff
@@ -0,0 +1,916 @@
+
+If the community cannot or does not want to use the a.DI or the other voting networks, then the community still can carry out the vote on Ethereum Mainnet. This would eliminate the cross-chain dependencies for voting.
+
+The cross-chain voting system has low centralisation risk because of its fault tolerant design, while the oracle system is currently unmitigated.
+
+> Autonomy score: High
+
+## Exit Window
+
+Critical permissions, including protocol upgrades, are controlled by the _Aave Governance_ onchain governance system.
+
+Two levels of permissions are separated in the (current) implementation of _Aave Governance_: _Level 2_ defines permissions on the governance system itself and on the `AAVE` token contract, _Level 1_ covers the remaining permissions.
+
+AAVE holders (also stkAAVE, aAAVE) are able to create new proposals (requires 80,000 / 200,0000 votes for level 1 / level 2) and vote on proposals (at least 320,000 / 1,040,000 votes are required for a valid proposal and a voting differential (yes minus no) of 80,000 / 1,040,000 votes).
+
+Votes start 1 day after proposal creation and the voting period is 10 days for _Level 2_, and 3 days for _Level 1_ proposals.
+
+Furthermore, the execution of successful votes is blocked by an _Exit Window_ with a delay of 7 days for _Level 2_ and 1 day for _Level 1_ permissions.
+
+Hence, the enforced _Exit Windows_ do not fulfill the 30-days delay required for a _Low_ risk assessment and only the _Level 2_ delay of 7 days qualifies for a _Medium_ risk according to our criteria.
+
+However, this is mitigated by the _Aave Governance_ system through the _Aave Governance V3 Guardian_ multisig account, which satisfies the Security Council requirements, and its ability to block unintended proposals from execution.
+
+> Exit Window score: High
+
+## Accessibility
+
+The official frontend to the Aave V3 protocol is open source and each commit on Aave's GitHub repository is published to IPFS (https://github.com/aave/interface/releases).
+
+In addition to that the Aave v3 protocol is also available through [DeFi Saver](https://app.defisaver.com/aave) and other third-party tools.
+
+https://aave.com/help/aave-101/accessing-aave
+
+> Accessibility score: Low
+
+## Conclusion
+
+The Aave-v3 Ethereum mainnet protocol achieves High centralization risk scores for its Upgradeability, Autonomy and Exit Window dimensions. It thus ranks **Stage 0**.
+
+The project mitigates the Exit Window risk by having a Governance Guardian that suffices the Security Council setup.
+
+The protocol could reach **Stage 1** by implementing fallback mechanism around the Chainlink oracle (or Chainlink adopting a Security Council setup for its own multisig account).
+
+The project additionally could advance to **Stage 2** if the on-chain governance (DAO) used a 30-day exit window instead of only 7 days.
```

Comment:
The project additionally could advance to **Stage 2** if all critical permissions were assigned to Aave's onchain governance system and protected with a 30-day _Exit Window_.